### PR TITLE
Issue #13: Validate redirects

### PR DESCRIPTION
**Pull request by lmorchard**
- Add redirects to the common schema
- Initial support for the `items` schema key for validating arrays
- Tests

### CODE REVIEW

**Comment by lmorchard** [comment]
@ednapiranha you might want to poke some holes in this. I added support for [`items` from JSON schema](https://github.com/mozilla/node-firefox-app-validator-manifest/pull/20/files#diff-58344f5126a890358efb47ad05cc2417R90) to validate arrays, but it might be funky yet.

File: rules/common.json [hunk 1]
```diff
@@ -87,7 +87,14 @@
 
         },
         "redirects": {
-
+            "type": "array",
+            "items": {
```
Comment (operatorjen) [inline rules/common.json:91]: i think this has to be wrapped by `properties` like so https://github.com/lmorchard/node-firefox-app-validator-manifest/blob/13-redirects/rules/common.json#L31

Comment (lmorchard) [inline rules/common.json:91]: I got this from [a JSON schema example](http://json-schema.org/example1.html). Looks like `properties` are for objects, but `items` are for arrays.

The "Are all tags strings?" part looks like this:

```
        "tags": {
            "type": "array",
            "items": {
                "type": "string"
            },
            "minItems": 1,
            "uniqueItems": true
        }
```

Comment (operatorjen) [inline rules/common.json:91]: ooooh ok i see.

**Pull request merged**